addon/evaluate-conditions.js

```diff
@@ -1,4 +1,11 @@
+/**
+ * @module evaluate-conditions
```

Comment:
Fair enough, but really consumers are free to import anything we export, and if we change what we export, we should probably do it in a friendly way. Particularly since this code might be handy for a custom renderer, I thought it'd be nice to go the deprecation route.